src/oop/programs_with_multiple_classes/hard/q4/QuestionText.md

```diff
@@ -0,0 +1,34 @@
+# oop-programs_with_multiple_classes-hard-q3
+
+You want to write a program that models books in a bookcase. To do this, create a Java program with two
+classes: `Book` and `Bookcase`. See the requirements below for each class.
+
+## Book Class
+
+Your `Book` class should have the following items:
+
+- Attributes:
+    - An `int` called **year**
+    - A `String` called **type**
+- Constructors:
+    - A constructor which accepts a String to initialize the **type** field and an int to initialize the **year**
+      field (in this order)
```

Comment:
Can you format this md file, please? You should be able to use ctrl - alt - l on it like any other file in Intellij